### PR TITLE
Text to describe set up that may be necessary to use the Emacs text editor

### PR DESCRIPTION
**Pull request by theabro**
Brief introduction to Emacs and how to set-up f90-mode for different file extensions, and where to place this depending on OS type.

### CODE REVIEW

File: learners/setup.md [hunk 1]
```diff
@@ -42,5 +42,33 @@ If you are not taking this course on archer2 we recommend installing the [GFortr
 
 :::::::::::::::::::::::::
 
+## Text Editors and Integrated Development Environments (IDEs)
 
+While Fortran files are plain text can could be edited with any plain text editor, it is recommened that you use an editor that is capable of _syntax highlighting_. There are a number of these available.
```
Comment (astroDimitrios) [inline learners/setup.md:47]:
```suggestion
While Fortran files are plain text and can be edited with any plain text editor, it is recommened that you use an editor that is capable of _syntax highlighting_. Setup instructions for some editors are available below.
```

File: learners/setup.md [hunk 2]
```diff
@@ -42,5 +42,33 @@ If you are not taking this course on archer2 we recommend installing the [GFortr
...
 
+### Emacs
+
+[Emacs][emacs-link] is a cross-platform text editor available for Linux, MacOS, and Windows. It will contain syntax highlighting for many Fortran file, but additional file extensions, such as `.X90`  may be valid Fortran files but will not be picked up in Emacs _f90-mode_. This mode allows for tab-completion of Fortran statements and the addition of a drop-down menu that provides additional Fortran-related options.
```
Comment (astroDimitrios) [inline learners/setup.md:51]:
```suggestion
[Emacs][emacs-link] is a cross-platform text editor available for Linux, MacOS, and Windows. Emacs applies syntax highlighting for most Fortran files, but additional file extensions, such as `.X90`  may be valid Fortran files but will not be picked up in Emacs _f90-mode_. This mode allows for tab-completion of Fortran statements and the addition of a drop-down menu that provides additional Fortran-related options.
```

**Review by astroDimitrios — CHANGES_REQUESTED** [review]
Thanks for this and for adding in the NCAS and emacs links! I have made a few suggested changes to the wording in places.

File: learners/setup.md (hunk 2 above)
Comment (theabro) [reply learners/setup.md:51]: Many thanks - I've committed the suggestion

**Comment by theabro** [comment]
Many thanks for the review - I've committed the suggested changes.

**Review by astroDimitrios — APPROVED** [review]
LGTM :)

**Pull request merged**